src/index.html

```diff
@@ -17,6 +17,86 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__link-logo"
+      >
+        <img
+          src="/img/logo-moyo.png"
+          alt="logo"
+          class="header__logo"
+        />
+      </a>
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active apple"
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. Consider renaming the class 'apple' to something more meaningful that describes the content or purpose of the link.